config/environment.rb

```diff
@@ -3,3 +3,15 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.delivery_method = :smtp
+ActionMailer::Base.default_url_options[:host] = "localhost:3000"
+ActionMailer::Base.smtp_settings = {
+  address: Rails.application.secrets.email_address,
+  port: 587,
+  user_name: Rails.application.secrets.email_user_name,
+  password: Rails.application.secrets.email_user_password,
+  authentication: :plain,
+  enable_starttls_auto: true,
+  openssl_verify_mode: 'none'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.